idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -13,35 +14,39 @@ export default PageView.extend({
     'click [data-hook=uninstall]': 'uninstall'
   },
 
+  bindings: {
+    'model.title': {
+      type: 'text',
+      hook: 'title'
+    },
+
+    'model.enabled': {
+      type: 'toggle',
+      yes: '[data-hook=uninstall]',
+      no: '[data-hook=install]'
+    }
+  },
+
   initialize(opts) {
     this.model = app.experiments.get(opts.slug, 'slug');
   },
 
   render() {
-    this.modified_date = new Date(this.model.modified);
-    this.created_date = new Date(this.model.created);
-
     // TODO: let's not mess with body, if possible
-
-    // TODO:(DJ) enabled property is not set here when navigating
-    // directly to the experiment page. This will not be a problem
-    // when we start persisting state with the User Installation model.
-    if (this.model.enabled) {
-      document.body.classList.add('active');
-    } else {
-      document.body.classList.add('inactive');
-    }
     document.body._id = document.body.id;
     document.body.id = 'idea-view';
 
     PageView.prototype.render.apply(this, arguments);
+
+    this.renderSubview(new DetailsView({
```

Comment:
FWIW, you can simplify this a bit by declaring something like this after `bindings`:
```
subviews: {
  details: {
    container: '.details-contain',
    constructor: DetailsView
  }
}
```
Then, `renderSubview` will get automatically called. Also, by default the constructor will get called with `{ parent: this }` from this view, and so `DetailsView` can access the parent model with `this.parent.model`. You can use that or [simplify the reference in the subview](https://github.com/lmorchard/c25k-web/blob/master/src/js/views/AudioCues.js#L9).